Notesnook.API/Controllers/MonographsController.cs

```diff
@@ -153,9 +161,13 @@ public async Task<IActionResult> UpdateAsync([FromBody] Monograph monograph)
                 );
                 if (!result.IsAcknowledged) return BadRequest();
 
+
+                SyncMonograph(monograph.ItemId ?? monograph.Id, deviceId);
+
                 return Ok(new
                 {
-                    id = monograph.ItemId
+                    id = monograph.ItemId,
+                    datePublished = monograph.DatePublished,
```

Comment:
Why?

Reply:
`datePublished` is defined in the server, we need to send it to the client so that the client can update accordingly. The field itself isn't used in the client somewhere, but we could need it in the future